src/alert/alert.scss

```diff
@@ -16,53 +17,64 @@
     color: t(iui-text-color);
     background-color: t(iui-color-background-1);
   }
-
-  &-icon {
-    @include iui-icons-default;
-    flex-shrink: 0;
-    margin-left: $iui-m;
-  }
-
-  &-message {
-    margin: $iui-baseline $iui-m;
-  }
-
-  &-link {
-    @include iui-anchor-underline('on-initial');
-    border-radius: $iui-border-radius;
-    cursor: pointer;
-    margin-left: $iui-s;
-    user-select: none;
-    white-space: nowrap;
-  }
-
-  > .iui-button {
-    margin-left: auto;
-    margin-right: $iui-s;
+  @each $status in positive, negative, warning, informational {
+    $statusColor: if($status == informational, 'primary', $status);
+    &.iui-#{$status} {
+      @include themed {
+        background: linear-gradient(
+          rgba(t(iui-color-foreground-#{$statusColor}-rgb), t(iui-opacity-6)),
+          rgba(t(iui-color-foreground-#{$statusColor}-rgb), t(iui-opacity-6))
+        ),
+          linear-gradient(t(iui-color-background-1), t(iui-color-background-1));
+        border-color: t(iui-color-foreground-#{$statusColor});
+      }
+    }
   }
 
   &.iui-sticky {
     border-radius: 0;
-    position: absolute;
+    position: fixed;
     top: 0;
     left: 0;
     width: 100vw;
   }
 }
+@mixin iui-alert-icon {
+  @include iui-icons-default;
+  flex-shrink: 0;
+  margin-left: $iui-m;
+}
+@mixin iui-alert-message {
+  margin: $iui-baseline $iui-m;
+}
+
+@mixin iui-alert-link {
+  @include iui-anchor-underline('on-initial');
+  border-radius: $iui-border-radius;
+  cursor: pointer;
+  margin-left: $iui-s;
+  user-select: none;
+  white-space: nowrap;
+}
+
+@mixin iui-alert-button {
+  @include iui-button;
+  @include iui-button-borderless;
+  @include iui-button-size(small, borderless);
+  margin-left: auto;
+  margin-right: $iui-s;
+}
+
+@mixin iui-alert-button-icon {
+  @include iui-button-icon;
+  margin-left: 3px;
```

Comment:
weird place.. maybe use `$iui-component-offset` variable and add comment, why it was needed.

Reply:
This has been updated